src/HeadTag.js

```diff
@@ -36,15 +36,23 @@ export default class HeadTag extends Component {
   render() {
     const { tag: Tag, ...rest } = this.props;
 
-    const Comp = <Tag {...rest} />;
-
     if (this.state.canUseDOM) {
+      const Comp = (
+        <Tag
+          key={`${Tag}${Object.keys(rest)
```

Comment:
Help me understand why do we need a key field?

Reply:
As far as I'm concerned they are only needed for SSR support.